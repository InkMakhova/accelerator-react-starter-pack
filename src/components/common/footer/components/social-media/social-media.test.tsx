import React from 'react';
import { render } from '@testing-library/react';
import SocialMedia from './social-media';

it('should render SocialMedia correctly', () => {
  const {container} = render(<SocialMedia />);
  expect(container).toMatchSnapshot();
});
