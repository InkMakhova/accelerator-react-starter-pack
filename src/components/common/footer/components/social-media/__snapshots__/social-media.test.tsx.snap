// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SocialMedia correctly 1`] = `
<div>
  <div
    class="socials footer__socials"
  >
    <ul
      class="socials__list"
    >
      <li
        class="socials-item"
      >
        <a
          aria-label="facebook"
          class="socials__link"
          href="https://www.facebook.com/"
        >
          <svg
            aria-hidden="true"
            class="socials__icon"
            height="24"
            width="24"
          >
            <use
              xlink:href="#icon-facebook"
            />
          </svg>
        </a>
      </li>
      <li
        class="socials-item"
      >
        <a
          aria-label="instagram"
          class="socials__link"
          href="https://www.instagram.com/"
        >
          <svg
            aria-hidden="true"
            class="socials__icon"
            height="24"
            width="24"
          >
            <use
              xlink:href="#icon-instagram"
            />
          </svg>
        </a>
      </li>
      <li
        class="socials-item"
      >
        <a
          aria-label="twitter"
          class="socials__link"
          href="https://www.twitter.com/"
        >
          <svg
            aria-hidden="true"
            class="socials__icon"
            height="24"
            width="24"
          >
            <use
              xlink:href="#icon-twitter"
            />
          </svg>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
